### ISSUE TITLE
Request: build monero-core binary artifacts

### ISSUE THREAD

**Issue by Jaqueeee** [monero-project/meta#14]
Windows binaries needs to be built without avx cpu flags to be supported on older machines. 
- static libboost build with -mno-avx and same toolchain as qt5 prebuilt mingw binaries. Inspiration here: https://forum.getmonero.org/5/support/2510/building-monero-v0-9-2-on-win32
Qt 5 mingw install: http://download.qt.io/official_releases/qt/5.6/5.6.2/qt-opensource-windows-x86-mingw492-5.6.2.exe
- libwallet built with same toolchain
- monero-core build

Other platforms can be built with official instructions found in Readme.

**Comment by danrmiller** [comment]
Once #160 is merged, download links for the .exe for the win32 monero-core builds should work.

I can't retrigger the build for 160 because I think buildbot was down when the original webhook came in.

I'll look at the boost and other special build requirements and give an update.

**Comment by anonimal** [comment]
Is this issue still applicable?

**Comment by danrmiller** [comment]
Download links for the win32 GUI binaries don't include a monero daemon like the downloads for all the other platforms do.

The win32 daemon that buildbot successfully builds does not run properly because msys2 g++ uses POSIX threads and DWARF exception handling on this platform. We can leave this ticket open, or I can start a new issue for setting up an environment using a compiler with Win32 native thread support AND setjmp/longjmp exception handling. 

As far as I'm aware, currently if you don't configure and build this yourself, the only binary available is https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/4.9.3/threads-win32/sjlj/i686-4.9.3-release-win32-sjlj-rt_v4-rev1.7z/download 

@Jaqueeee 

> Windows binaries needs to be built without avx cpu flags to be supported on older machines.

Is this handled by the build.sh script in the monero-core repo?

> static libboost build with -mno-avx and same toolchain as qt5 prebuilt mingw binaries. Inspiration here: https://forum.getmonero.org/5/support/2510/building-monero-v0-9-2-on-win32
Qt 5 mingw install: http://download.qt.io/official_releases/qt/5.6/5.6.2/qt-opensource-windows-x86-mingw492-5.6.2.exe

What is the current suggestion for QT on win32? I think this might have changed since this issue was opened. I know we are currently using a 5.7 build, but I'm not sure the configuration.

**Comment by danrmiller** [comment]
@Jaqueeee are we all set on tester builds for the GUI for now?

**Comment by Jaqueeee** [comment]
@danrmiller yes!
